pytket/extensions/quantinuum/backends/quantinuum.py

```diff
@@ -451,7 +451,7 @@ def get_calendar(
         start_date: datetime.datetime,
         end_date: datetime.datetime,
         localise: bool = True,
-    ) -> List[Dict[str, str]]:
+    ) -> List[Dict[str, object]]:
```

Comment:
I think it is better to use `Any` here. See e.g. https://stackoverflow.com/questions/39817081/typing-any-vs-object